hiplan/base/app.py

```diff
@@ -33,3 +33,11 @@ def serialize(self):
     def save_to_file(self, file_name:str = "App.json"):
         with open(file_name, "w") as f:
             f.write(json.dumps(self.serialize()))
+
+    @staticmethod
+    def deserialize(dict):
+        App.__instance.boards = []
```

Comment:
`__instance` este un membru privat pe care nu ar trebui sa-l accesezi asa. Asta si s-ar putea sa fie None, caz in care ti-ar da eroare. Foloseste `get_instance` daca ai nevoie de instanta aplicatiei